api/controllers/loomies.go

```diff
@@ -167,3 +168,33 @@ func HandleNearLoomies(c *gin.Context) {
 		"loomies": wildLoomies,
 	})
 }
+
+func HandleValidateLoomieExists(c *gin.Context) {
+	var loomie_id interfaces.LoomieExistsForm
+
+	if err := c.BindJSON(&loomie_id); err != nil {
+		fmt.Println(err)
+		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": true, "message": "Bad request"})
+		return
+	}
+
+	if loomie_id.LoomieId == "" {
+		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": true, "message": "No Loomie ID"})
+		return
+	}
+
+	exists, _ := models.ValidateLoomieExists(loomie_id.LoomieId)
+
+	if !exists {
+		c.AbortWithStatusJSON(http.StatusNotFound, gin.H{
```

Comment:
Please, follow this schema for the responses on ALL the endpoints: 

```json
{
  "error": "boolean:", 
  "message": "string"
}
```

You can add other fields if needed, but always include the `error` and `message` fields.